content/docs/command-reference/commit.md

```diff
@@ -40,12 +40,10 @@ scenarios are further detailed below.
   reproduce the whole pipeline. If you're sure no pipeline results would change,
   use `dvc commit` to force update the `dvc.lock` or `.dvc` files and cache.
 
-- In cases where we have previously executed a stage (either by writing
-  `dvc.yaml` manually and using `dvc repro`, or with `dvc run`), but later
-  notice that some of the output files or directories it creates, which are
-  already in the <abbr>workspace</abbr>, are missing from `dvc.yaml` (`outs`
-  field). We can
-  [add missing outputs to an existing stage](/docs/user-guide/how-to/add-output-to-stage)
+- In cases where we have previously executed a stage and later notice that some
+  of the files/directories used by the stage as dependencies, or created as
+  outputs, are missing from `dvc.yaml`. We can
+  [add missing dependencies/outputs to an existing stage](/docs/user-guide/how-to/add-deps-or-outs-to-a-stage)
   without having to execute it again. Use `dvc commit` to update the `dvc.lock`
   file and save outputs to the cache.
```

Comment:
Per https://github.com/iterative/dvc.org/pull/1914#pullrequestreview-533883376:
```suggestion
- In some cases, we have previously executed a stage, and later notice that some
  of the files/directories used by the stage as dependencies or created as
  outputs are missing from `dvc.yaml`. It is possible to
  [add missing data to an existing stage](/docs/user-guide/how-to/add-deps-or-outs-to-a-stage).
  Use `dvc commit` to update the `dvc.lock` file and save outputs to the cache.
```
Please double check the text and format it if needed.

Reply:
Done